packages/docusaurus-theme-redoc/src/theme/ApiSchema/ApiSchema.tsx

```diff
@@ -3,20 +3,17 @@ import clsx from 'clsx';
 import { ThemeProvider } from 'styled-components';
 import '../../global';
 import { SchemaDefinition } from 'redoc';
-import { useSpec } from '../../utils/useSpec';
-import { useSpecData } from '../useSpecData';
+import useSpec from '../../utils/useSpec';
 import { ApiSchemaProps as Props } from '../../types/common';
 import '../Redoc/styles.css';
 import './styles.css';
 
 const ApiSchema: React.FC<Props> = ({
-  id,
-  example,
+  showExample,
```

Comment:
I don't want to completely remove the old id options as it would require lots of people to migrate.

Can you add support for both the old props and the new ones? It can even be a new component if supporting both is a problem.

Reply:
`id` is still working of course. It's still present inside `rest` object

Reply:
fixed - it's easier to see that id and spec are supported